migrations/sql/V2021.08.20.16.09__create_itrb_exemption_code_type_codes.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS itrb_exemption_status (
+    itrb_exemption_status_code       varchar                                NOT NULL PRIMARY KEY,
+    description                      varchar                                NOT NULL            ,
+    active_ind                       boolean                  DEFAULT true  NOT NULL            ,
+    create_user                      varchar                                NOT NULL            ,
+    display_order					 smallint                               NOT NULL            ,
+    create_timestamp                 timestamp with time zone DEFAULT now() NOT NULL            ,
+    update_user                      varchar                                NOT NULL            ,
+    update_timestamp                 timestamp with time zone DEFAULT now() NOT NULL
+);
+
+
+ALTER TABLE itrb_exemption_status OWNER TO mds;
+
+INSERT INTO itrb_exemption_status
+    (
+    itrb_exemption_status_code,
+    description,
+    display_order,
+    create_user,
+    update_user
+    )
+VALUES
+    ('NO', 'No', 10, 'system-mds', 'system-mds'),
+    ('YES', 'Yes', 20, 'system-mds', 'system-mds'),
+    ('EXEM', 'Exemption', 30, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
+
+ALTER TABLE mine_tailings_storage_facility ADD COLUMN IF NOT EXISTS itrb_exemption_status_code varchar REFERENCES itrb_exemption_status(itrb_exemption_status_code);
+
+UPDATE mine_tailings_storage_facility SET itrb_exemption_status_code = 'YES' WHERE has_itrb = 'true';
```

Comment:
I don't believe we should set a default since we changed the meaning of the field. Null is fine. (I discussed this change with Aaron also) 

Sorry for the confusion on the ticket, We will make sure they are more clear in the future.

Reply:
fixed.

Reply:
I still see the defaults being set, was that change not pushed up yet?